packages/fxa-auth-server/lib/routes/utils/cms/localization.ts

```diff
@@ -188,35 +190,92 @@ export class CMSLocalization {
       ftlLines.push(`${entry.ftlId} = ${entry.value}`);
     }
 
-    return ftlLines.join('\n');
+    const ftlContent = ftlLines.join('\n');
+    return this.sanitizeFtlContent(ftlContent);
+  }
+
+  /**
+   * Sanitize entire FTL content to remove hidden Unicode characters
```

Comment:
The comment doesn't match the function here. I think the comment should be updated (vs the function), but prob good to confirm.